js/live-editor.js

```diff
@@ -166,15 +138,25 @@ window.LiveEditor = Backbone.View.extend({
         }
 
         this.bind();
+
+        // jquery.ui.slider.js is require for the toolbar and audio to work
+        
+        // don't set up the toolbar if we don't need to
+        // this.setupToolbar();
+
+        // this should only be loaded at startup if we know that the program
+        // being loaded has a sound in it
+        // if not, we should be able to load this on demand when a person
+        // types "getSound"
         this.setupAudio();
     },
 
     render: function() {
-        this.$el.html(Handlebars.templates["live-editor"]({
-            execFile: this.execFile,
-            imagesDir: this.imagesDir,
-            colors: this.colors
-        }));
+        //this.$el.html(Handlebars.templates["live-editor"]({
+        //    execFile: this.execFile,
```

Comment:
This should just be removed now that the HTML for live-editor is directly in index.html.  I'm curious though about how live-editor gets packaged for production.  The demo/simple/index.html isn't used in production, is it?  If that's the case, we'd have to make sure that the HTML that's in index.html actually makes its way into production.